tedana/metrics/collect.py

```diff
@@ -119,7 +119,10 @@ def generate_metrics(
     metric_maps = {}
     if "map weight" in required_metrics:
         LGR.info("Calculating weight maps")
-        metric_maps["map weight"] = dependence.calculate_weights(data_optcom, mixing)
+        metric_maps["map weight"] = dependence.calculate_weights(
+            data_optcom=data_optcom,
```

Comment:
Related to https://github.com/ME-ICA/tedana/pull/1086#discussion_r1574904654- we use `data_oc` in other places. I'd like to commit to one or the other and make the variable names consistent between this PR and #1086.

Reply:
I personally like `data_optcom` better.

Reply:
That works for me. @handwerkerd do you have an opinion on this?

Reply:
No opinion.